index.js

```diff
@@ -86,6 +107,16 @@ var clean = function() {
     webpackDir = upperCaseWindowsRoot(webpackDir);
   }
 
+  // Resolve RegExp paths.
+  _this.paths.reduce((acc, currentPath) => {
```

Comment:
You should be assigning the result of this `.reduce` back again to `_this,paths`. Better else, You can instead map the regex to resolved paths and assign it to `_this.paths`

Reply:
Hey, okay I'll look into it!